server/system/rest.yaml

```diff
@@ -2256,3 +2256,34 @@ endpoints:
             required: false
             title: Exclude (0, default), include (1) or return only (2) deleted connections
             type: filter.State
+
+- title: SMTP Connection Checker
+  entrypoint: smtpConnection
+  path: "/smtp/connection"
+  apis:
+    - name: check
+      method: POST
+      title: Check SMTP server settings
+      path: "/checker/"
+      parameters:
+        post:
+          - type: string
+            name: host
+            required: true
+            title: SMTP server host name
+          - type: uint
+            name: port
+            required: true
+            title: SMTP server port
+          - type: "[]string"
+            name: recipients
+            required: false
+            title: List of recipients email addresses that should recieve test email
+          - type: string
+            name: username
+            required: false
+            title: SMTP server authentication username
+          - type: string
+            name: password
+            required: false
```

Comment:
remove `required: false` (this is default) and if you can, squeeze each param into a single line so the whole thing is a bit shorter (Se "Create client" in the same file for example